.github/autobuild/mac.sh

```diff
@@ -146,6 +180,27 @@ pass_artifact_to_job() {
     echo "Moving build artifact to deploy/${artifact}"
     mv ./deploy/Jamulus-*installer-mac.dmg "./deploy/${artifact}"
     echo "artifact_1=${artifact}" >> "$GITHUB_OUTPUT"
+
+    artifact2="jamulus_${JAMULUS_BUILD_VERSION}_mac${ARTIFACT_SUFFIX:-}.pkg"
+    for file in ./deploy/Jamulus_*.pkg; do
+        if [ -f "${file}" ]; then
+            echo "Moving build artifact2 to deploy/${artifact2}"
+            mv "${file}" "./deploy/${artifact2}"
+            echo "artifact_2=${artifact2}" >> "$GITHUB_OUTPUT"
```

Comment:
May be buggy to write/overwrite artifact_2 in a loop.
There's probably a more efficient approach!

Reply:
I think it's ok. As far as I can tell, there will be either zero or one `Jamulus_*.pkg` files in `deploy/`. In either case, the loop will only go round once, with `file` set either to the pattern (if it didn't match) or the filename (if it did match). In the former case, the `if [ -f` will fail and nothing will be done. In the latter case, the artifact will be moved and the output set.

It could probably be made clearer this way:
```
file=./deploy/Jamulus_*.pkg
if [ -f "${file}" ]; then
    echo "Moving build artifact2 to deploy/${artifact2}"
    mv "${file}" "./deploy/${artifact2}"
    echo "artifact_2=${artifact2}" >> "$GITHUB_OUTPUT"
fi
```
(the assignment to `file` will expand the glob if quotes are not used)

Reply:
The glob is just a shorthand to avoid calculating the complete filename, just as in the `mv` of `${artifact}` a few lines above.

Reply:
I've just tried my suggestion in a test branch. Shellcheck doesn't like the line `file=./deploy/Jamulus_*.pkg`. I suggest we leave the code as it is with the `for`.

Reply:
OK, Shellcheck is happy with this code:
```
    file=(./deploy/Jamulus_*.pkg)
    if [ -f "${file[0]}" ]; then
        echo "Moving build artifact2 to deploy/${artifact2}"
        mv "${file[0]}" "./deploy/${artifact2}"
        echo "artifact_2=${artifact2}" >> "$GITHUB_OUTPUT"
    fi
````